functions/greater-than-ten/greater.js

```diff
@@ -0,0 +1,23 @@
+/*function checkValue(highNumber) {
+  if (highNumber > 10) {
+    return true;
+  } else {
+    return "sorry low number";
+  }
+}
+highNumber = 12;
+console.log(highNumber);
+alert(highNumber);
+*/
+
+checkValue = x => {
+  if (x > 10) {
+    return "15 is greater than 10";

```

Comment:
function should return if the number is greater then 10, ie a boolean (true | flase)